ts/receiver/contentMessage.ts

```diff
@@ -343,11 +345,21 @@ export async function innerHandleSwarmContentMessage(
     const content = SignalService.Content.decode(new Uint8Array(plaintext));
     perfEnd(`SignalService.Content.decode-${envelope.id}`, 'SignalService.Content.decode');
 
-    const blocked = await isBlocked(envelope.source);
+    /**
+     * senderIdentity is set ONLY if that message is a closed group message.
+     * If the current message is a closed group message,
+     * envelope.source is going to be the real sender of that message.
```

Comment:
So in a closed group the senderIdentity is the admin or member and the source is the closed group key? I'm a bit confused?

Reply:
updated comment

Reply:
👍